### DISCUSSION TITLE
Initial mode of "Easy Effects" at startup: Output/Input

### DISCUSSION THREAD

**Discussion by hayama77** [wwmm/easyeffects#3221] [Q&A]
I want "Input" and "Effect" to be selected whenever Easy Effects starts up, but how can I do that? 
Currently, "Output" and "Player" are always selected as default. 
Can I change that by modifying the configuration file?

**Comment by wwmm** [comment]
At this moment there is not configuration option for this. But it should not be too hard to do. Feel free to open an issue requesting this feature.

**Reply by hayama77** [reply]
Thank you. 
You can set it in the configuration file, or you can have it remember the previous state of 'Easy Effects' and start it from that state the next time you start it.   (^_^
